src/data-tutorials/users.tsx

```diff
@@ -957,6 +957,20 @@ const Users: User[] = [
     website: 'blog/tutorials/truffle-and-celo-the-ultimate-guide-to-deploy-celo-dapps-with-truffle',
     tags: ['beginner','foundation', 'truffle'],
   },
+  {
+    title: 'Exploring Celo in the Gaming Industry for In-Game Assets and P2P Trading',
```

Comment:
Why this is being changed in this document?

Reply:
I saw the user.txt file had a conflict, thought it was because i deleted that section when pushing this tutorial to another branch.

Reply:
Ok